mppsolar/protocols/pi30m054.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+from .pi30m045 import pi30m045
+
+log = logging.getLogger("pi30m054")
+
+QUERY_COMMANDS = {
+    "QBMS": {
+        "name": "QBMS",
+        "description": "Read lithium battery information",
+        "help": " -- queries the value of various metrics from the battery",
+        "type": "QUERY",
+        "crctype": "chk",
+        "response": [
+            ["keyed", "Battery connect status", {"0": "Connected", "1": "Disconnected"}],
+            ["int", "Battery capacity from BMS", "%"],
+            ["keyed", "Battery force charging", {"0": "No", "1": "Yes"}],
+            ["keyed", "Battery stop discharge flag", {"0": "Enable discharge", "1": "Disable discharge"}],
+            ["keyed", "Battery stop charge flag", {"0": "Enable charge", "1": "Disable charge"}],
+            ["int", "Battery bulk charging voltage from BMS", "0.1 V"],
```

Comment:
@jblance I tried what you suggested on #402 with `int:r*10`, vut if ends on:
```python
  File "mppsolar/outputs/screen.py", line 60, in output
    key = key.replace(" ", "_")
          ^^^^^^^^^^^
AttributeError: 'int' object has no attribute 'replace'
```

for the sake of completness, I tried the `json` output and got the same error. Do you have an idea how to workaround that?

Reply:
it needs to be the INDEXED format 
```python
"QBMS": {
        "name": "QBMS",
        "description": "Read lithium battery information",
        "help": " -- queries the value of various metrics from the battery",
        "type": "QUERY",
        "crctype": "chk",
        "response_type": "INDEXED",
        "response": [
            [0, "Battery connect status", "str_keyed", {"0": "Connected", "1": "Disconnected"}],
            [1, "Battery capacity from BMS", "int", "%"],
            [2, "Battery force charging", "str_keyed", {"0": "No", "1": "Yes"}],
            [3, "Battery stop discharge flag", "str_keyed", {"0": "Enable discharge", "1": "Disable discharge"}],
            [4, "Battery stop charge flag", "str_keyed", {"0": "Enable charge", "1": "Disable charge"}],
            [5, "Battery bulk charging voltage from BMS", "int:r/10", "V"],
            [6, "Battery float charging voltage from BMS", "int:r/10", "V"],
            [7, "Battery cut off voltage from BMS", "int:r/10", "V"],
            [8, "Battery max charging current", "float", "A"],
            [9, "Battery max discharge current", "float", "A"]],
        "test_responses": [
            b"(0 086 0 0 0 532 532 450 0120 0240\x7fU\r",
        ],
    },
```

Reply:
Awesome, I tried several things but not that. Thanks a lot!